seed/challenges/02-javascript-algorithms-and-data-structures/es6.json

```diff
@@ -454,16 +453,14 @@
       "challengeSeed": [
         "let a = 8, b = 6;",
         "/* Alter code below this line */",
-        "a = b;",
-        "b = a;",
         "/* Alter code above this line */",
         "console.log(a); // should be 6",
         "console.log(b); // should be 8"
       ],
       "tests": [
-        "// Test a is 6",
-        "// Test b is 8",
-        "// Test destructuring was used"
+        "assert(a === 6, 'message: Value of <code>a</code> should be 6, after swapping.');",
+        "assert(b === 8, 'message: Value of <code>b</code> should be 8, after swapping.');",
+        "assert(/\\[\\s*.{1}\\s*,\\s*.{1}\\s*\\];/g.test(code), 'message: Use array destructuring to swap a and b.');"
```

Comment:
I made some changes to the regex to make sure it tests for a valid swap. I also removed the semicolon as it technically isn't required by JS.

![destructuring regex](https://cloud.githubusercontent.com/assets/6125097/23101246/5ce05f7c-f68f-11e6-9b00-e3a28049dcdd.png)